frontend/src/features/login/Login.js

```diff
@@ -33,7 +33,7 @@ const Login = () => {
   const formSubmit = (e) => {
     dispatch(reset());
     setError("");
-    setHide(true)
+    setHide(true);
```

Comment:
Just wondering if this semi colon missing here before still allowed you to npm build?

Reply:
yes it did otherwise i would get an error